packages/saber/vue-renderer/app/router.js

```diff
@@ -35,6 +36,7 @@ export default () => {
     new Router({
       mode: 'history',
       routes,
+      base: publicUrl,
```

Comment:
only needed in production?

Reply:
That's a non-issue, as `validateConfig` already takes care of that:

https://github.com/saberland/saber/blob/98ae9198ea210630a517e4945216df743f2ceb5a/packages/saber/lib/utils/validateConfig.js#L102-L105